### PR TITLE
Add sparse pool configurations for pre release yaml.

### PR DESCRIPTION
**Pull request by sonasingh46**
This commit will do following:
1. Add storagepoolclaim for sparse pool.
2. Add storageclass for sparse pool.

Signed-off-by: sonasingh46 <sonasingh46@gmail.com>

### CODE REVIEW

File: k8s/openebs-pre-release-features.yaml [hunk 1]
```diff
@@ -2009,3 +2009,32 @@ metadata:
 spec:
   path: "/var/openebs"
 ---
+apiVersion: openebs.io/v1alpha1
+kind: StoragePoolClaim
+metadata:
+  name: cstor-sparse-pool-0.7.0
```
Comment (kmova) [inline k8s/openebs-pre-release-features.yaml]: remove -0.7.0

File: k8s/openebs-pre-release-features.yaml [hunk 2]
```diff
@@ -2009,3 +2009,32 @@ metadata:
...
+  annotations:
+    cas.openebs.io/create-pool-template: cstor-pool-create-default-0.7.0
+    cas.openebs.io/delete-pool-template: cstor-pool-delete-default-0.7.0
+spec:
+  name: cstor-sparse-pool-0.7.0
```
Comment (kmova) [inline k8s/openebs-pre-release-features.yaml]: remove 0.7.0.

File: k8s/openebs-pre-release-features.yaml [hunk 3]
```diff
@@ -2009,3 +2009,32 @@ metadata:
...
+  maxPools: 3
+  poolSpec:
+    poolType: striped
+    cacheFile: /tmp/cstor-sparse-pool-0.7.0.cache
+    overProvisioning: false
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-cstor-sparse-0.7.0
```
Comment (kmova) [inline k8s/openebs-pre-release-features.yaml]: remove -- 0.7.0

File: k8s/openebs-pre-release-features.yaml [hunk 4]
```diff
@@ -2009,3 +2009,32 @@ metadata:
...
+  annotations:
+    cas.openebs.io/create-volume-template: cstor-volume-create-default-0.7.0
+    cas.openebs.io/delete-volume-template: cstor-volume-delete-default-0.7.0
+    cas.openebs.io/read-volume-template: cstor-volume-read-default-0.7.0
+    cas.openebs.io/config: |
+      - name: StoragePoolClaim
+        value: "cstor-sparse-pool-0.7.0"
```
Comment (kmova) [inline k8s/openebs-pre-release-features.yaml]: remove .. -0.7.0

File: k8s/openebs-pre-release-features.yaml [hunk 5]
```diff
@@ -230,6 +251,10 @@ spec:
               # From host, dir called /var/openebs/shared-<uid> is created to avoid clash if two replicas run on same node.
               path: /var/openebs/shared-a2b
               type: {{ .Config.HostPathType.value }}
+          - name: sparse
+            hostPath:
+              path: /var/openebs
+              type: Directory
```
Comment (kmova) [inline k8s/openebs-pre-release-features.yaml]: Can you change the type to : {{ .Config.HostPathType.value }}

Comment (sonasingh46) [reply k8s/openebs-pre-release-features.yaml]: Yes I will do it.

**Comment by sonasingh46** [comment]
I will raise a new PR addressing these changes.

**Pull request merged**